### ISSUE TITLE
Tabbing support

### ISSUE THREAD

**Issue by psychemedia** [innovationOUtside/nbev3devsim#117]
Keyboard shortcuts currently work in the simulator widget if the mouse cursor is over the widget. Ideally, a user should be able to tab to the widget, use the keyboard shortcuts, then tab back to the notebook.
 
Immediate question is: how do we make things tab-able at least at the level of tabbing from notebook to widget and making widget keyboard commands work, then tab back from widget to notebook to make notebook keyboard commands work (they have overlapping keyboard shortcuts…)
 
A secondary question is tabbing within the widget, but that is complicated by the large number of controls, some of which are nested in the visual UI, and that nesting  sense of `hidden’ controls should probably be respected in a tab UI); I would value discussing that with someone maybe for 21B presentation.
